### ISSUE TITLE
The pt_BR language is not working

### ISSUE THREAD

**Issue by LeandroStanger** [flathub/br.com.jeanhertel.adriconf#2]
![captura de tela de 2018-09-14 20-20-23](https://user-images.githubusercontent.com/26097419/45579152-13978800-b85c-11e8-8995-fdafe0277d0f.png)
```
Gtk-Message: 20:18:52.647: Failed to load module "canberra-gtk-module"
Gtk-Message: 20:18:52.675: Failed to load module "canberra-gtk-module"
Current language code is pt
Exception caught: Could not create parser context

Driver 'i965' doesn't support option 'always_have_depth_buffer' on application 'Topogun (32-bit)'. Option removed.
Driver 'i965' doesn't support option 'always_have_depth_buffer' on application 'Topogun (64-bit)'. Option removed.
Driver 'i965' doesn't support option 'glsl_correct_derivatives_after_discard' on application 'Rocket League'. Option removed.
Driver 'i965' doesn't support option 'glsl_correct_derivatives_after_discard' on application 'The Witcher 2'. Option removed.
Driver 'i965' doesn't support option 'mesa_glthread' on application 'Alien Isolation'. Option removed.
Driver 'i965' doesn't support option 'mesa_glthread' on application 'BioShock Infinite'. Option removed.
Driver 'i965' doesn't support option 'mesa_glthread' on application 'Borderlands 2'. Option removed.
Driver 'i965' doesn't support option 'mesa_glthread' on application 'Civilization 5'. Option removed.
Driver 'i965' doesn't support option 'mesa_glthread' on application 'Civilization 6'. Option removed.
Driver 'i965' doesn't support option 'mesa_glthread' on application 'Dreamfall Chapters'. Option removed.
Driver 'i965' doesn't support option 'mesa_glthread' on application 'Hitman'. Option removed.
Driver 'i965' doesn't support option 'mesa_glthread' on application 'Renowned Explorers: International Society'. Option removed.
Driver 'i965' doesn't support option 'mesa_glthread' on application 'Saints Row 2'. Option removed.
Driver 'i965' doesn't support option 'mesa_glthread' on application 'Saints Row: The Third'. Option removed.
Driver 'i965' doesn't support option 'mesa_glthread' on application 'Saints Row IV'. Option removed.
Driver 'i965' doesn't support option 'mesa_glthread' on application 'Saints Row: Gat out of Hell'. Option removed.
Driver 'i965' doesn't support option 'mesa_glthread' on application 'Sid Meier's: Civilization Beyond Earth'. Option removed.
Driver 'i965' doesn't support option 'mesa_glthread' on application 'American Truck Simulator'. Option removed.
Driver 'i965' doesn't support option 'mesa_glthread' on application 'Euro Truck Simulator 2'. Option removed.
Driver 'i965' doesn't support option 'mesa_glthread' on application 'Overlord'. Option removed.
Driver 'i965' doesn't support option 'mesa_glthread' on application 'Overlord 2'. Option removed.
Driver 'i965' doesn't support option 'mesa_glthread' on application 'War Thunder'. Option removed.
Driver 'i965' doesn't support option 'mesa_glthread' on application 'War Thunder (Wine)'. Option removed.
Driver 'i965' doesn't support option 'mesa_glthread' on application 'Outlast'. Option removed.
Driver 'i965' doesn't support option 'mesa_glthread' on application 'Mount and Blade Warband'. Option removed.
Driver 'i965' doesn't support option 'mesa_glthread' on application 'Medieval II: Total War'. Option removed.
Driver 'i965' doesn't support option 'mesa_glthread' on application 'Carnivores: Dinosaur Hunter Reborn (wine)'. Option removed.
Driver 'i965' doesn't support option 'mesa_glthread' on application 'Far Cry 2 (wine)'. Option removed.

```

**Comment by jlHertel** [comment]
In general most translation come directly from mesa, so things like "Performance" and "Image Quality" cannot be fixed by us.
But the output on the terminal is also in english, which in this case really indicates a problem.
Have you built and installed the translations?
If yes, could you please paste here a full output of your installation commands and the output of each of them?

**Comment by LeandroStanger** [comment]
@jlHertel I installed it through the terminal
>flatpak install flathub br.com.jeanhertel.adriconf

**Comment by jlHertel** [comment]
Hey @nedrichards 
Do you have any idea why the translations are not showing up?
Compiling from sources everything is installed correctly.
Also, where exactly does flatpak stores translations file?
Under normal apps it is usually `/usr/share/locale/`

**Comment by jlHertel** [comment]
This should have been solved with the latest release.
Can you kindly test and give feedback?

**Comment by LeandroStanger** [comment]
@jlHertel Não funcionou
![captura de tela de 2019-01-27 19-58-01](https://user-images.githubusercontent.com/26097419/51807444-90545d00-2265-11e9-8343-648691c1a4ed.png)

**Comment by jlHertel** [comment]
This is now finally solved with v2.3.
